src/AppKit/NSAccessibility.cs

```diff
@@ -48,11 +52,11 @@ public static CGRect GetFrameInView (NSView parentView, CGRect frame)
 		[Mac (10, 10)]
 #endif
 		[DllImport (Constants.AppKitLibrary)]
-		static extern CGPoint NSAccessibilityPointInView (NSView parentView, CGPoint point);
+		static extern CGPoint NSAccessibilityPointInView (NativeHandle parentView, CGPoint point);
 
 		public static CGPoint GetPointInView (NSView parentView, CGPoint point)
 		{
-			return NSAccessibilityPointInView (parentView, point);
+			return NSAccessibilityPointInView (parentView.Handle, point);
```

Comment:
This avoids a potential NullReferenceException:

```suggestion
			return NSAccessibilityPointInView (parentView.GetHandle (), point);
```